vizro-core/examples/features/app.py

```diff
@@ -68,7 +68,7 @@
 
                 Standard predefined actions are made available including **export data** and **filter interactions**.
                 """,
-            href="/export-data",
+            href="Export data",
```

Comment:
Is this the right capitalisation? I know it will convert to lowercase either way but would be nice to match exactly with the `Page.title` ("Export Data" or "Export data" - I don't know which it is).

Reply:
Yes, it's the right capitalisation - I was also trying to sync it 😄 So the page.title is actually "Export data"